json-webhook/.env

```diff
@@ -0,0 +1,8 @@
+# description: The URL for your webhook
```

Comment:
Having the same description for the two webhooks is a little confusing. It Might be nice to differentiate these two more clearly. I realize 
```
# description: The path to the webhook
# configurable: false
TWILIO_SMS_WEBHOOK_URL=/event-to-webhook
```
is what we wrote in the existing apps, but since its a private value only for quick deploy, it might be worth saying as much while being a little more descriptive that the other webhook is where the function is going to be calling